Extension/package.json

```diff
@@ -71,7 +70,7 @@
           {
             "id": "verify.compiler.mac",
             "title": "Check for a C++ compiler",
-            "description": "The C++ extension requires a C++ compiler to be installed. If you don't have one, you can install one using the instructions to the right. \n[Check for C++ compilers](command:C_Cpp.OpenCompilerQuickpick)",
+            "description": "The C++ extension requires a C++ compiler to be installed. If you don't have one, you can install one using the instructions to the right. \n[Check for C++ compilers](command:C_Cpp.SelectDefaultCompiler)",
```

Comment:
The link label should be "Select your default compiler" or something like that instead of "Check for C++ compilers" since this doesn't run the command that "checks" for compilers anymore.

Reply:
That might make the button text more accurately reflect the original purpose of the UI it triggers, but I don't think it would make the UI more intuitive.  In a way, it's showing you a list of compilers, just like "Check for" used to, with the addition of allowing you to pick one to use as the default.  But it also may not be clear to the user why they are being asked to pick one.  The title of the step should be update too, etc..  I think this needs more thought.